static/app/app/list/views.js

```diff
@@ -33,6 +34,15 @@ var LunchMoveView = Marionette.ItemView.extend({
         channel.trigger('edit');
         return false;
     },
+    deleteMove: function(e) {
+        e.preventDefault();
+        channel.request('entities:move').destroy({
+            success: function(){
+                channel.request('entities:move', { replace: true });
```

Comment:
Instead of building this into the request handler, I think it would be clear if you did the resetting inline here, using [Backbone.Model.clear](http://backbonejs.org/#Model-clear). By adding it as a separate method, it adds complexity to the methods by:
 - forcing you to have to look another place to see what something is actually doing 
 - making it more difficult the `entities:move` request because it now accepts an additional option with alternative behavior
If we were using the `replace` option repeatedly throughout the code, adding it in as an option would be definitely be good because it would allow for code re-use, but as is, it's unnecessary